packages/salesforcedx-vscode-replay-debugger/package.json

```diff
@@ -0,0 +1,124 @@
+{
+  "name": "salesforcedx-vscode-replay-debugger",
+  "displayName": "Apex Replay Debugger for Visual Studio Code",
+  "description": "Replay Apex execution from Apex Debug Log",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
+  "bugs": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode/issues"
+  },
+  "repository": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode"
+  },
+  "icon": "images/icon.png",
+  "galleryBanner": {
+    "color": "#ECECEC",
+    "theme": "light"
+  },
+  "version": "41.12.0",
+  "publisher": "salesforce",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.19.0"
+  },
+  "categories": ["Debuggers"],
+  "dependencies": {
+    "@salesforce/salesforcedx-utils-vscode": "41.12.0",
+    "vscode-debugadapter": "1.25.0",
+    "vscode-debugprotocol": "1.25.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.0.0",
+    "@types/mocha": "2.2.38",
+    "@types/node": "^6.0.40",
+    "@types/sinon": "^2.3.2",
+    "chai": "^4.0.2",
+    "mocha": "3.2.0",
+    "sinon": "^2.3.6",
+    "nyc": "^11.0.2",
+    "cross-env": "5.0.4",
+    "mocha-junit-reporter": "^1.13.0",
+    "mocha-multi-reporters": "^1.1.4",
+    "typescript": "2.6.2",
+    "vscode": "1.1.10",
+    "vscode-debugadapter-testsupport": "1.25.0"
+  },
+  "extensionDependencies": ["salesforce.salesforcedx-vscode-core"],
+  "scripts": {
+    "vscode:prepublish": "npm prune --production",
+    "vscode:package": "vsce package",
+    "compile": "tsc -p ./",
+    "lint": "tslint --project .",
+    "watch": "tsc -watch -p .",
+    "clean":
+      "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
+    "postinstall": "node ./node_modules/vscode/bin/install",
+    "test":
+      "./node_modules/.bin/cross-env VSCODE_NLS_CONFIG={} ./node_modules/.bin/nyc ./node_modules/.bin/_mocha --recursive out/test --reporter mocha-multi-reporters",
+    "test:unit":
+      "./node_modules/.bin/cross-env VSCODE_NLS_CONFIG={} ./node_modules/.bin/nyc ./node_modules/.bin/_mocha --recursive out/test/unit --reporter mocha-multi-reporters"
+  },
+  "activationEvents": [
+    "onDebug",
+    "onCommand:extension.replay-debugger.getLogFileName"
+  ],
+  "main": "./out/src",
+  "contributes": {
+    "breakpoints": [
+      {
+        "language": "apex"
+      }
+    ],
+    "debuggers": [
+      {
+        "type": "apex-replay",
+        "label": "Apex Replay Debugger",
```

Comment:
This label is shown in the dropdown of debuggers when you try to create a new launch config.

![image](https://user-images.githubusercontent.com/7286437/34685584-b233f840-f45d-11e7-8e0f-8311b95510d1.png)

Reply:
Should we be more specific about the debuggers now and say "Apex Interactive Debugger"?

Reply:
The official name of the existing debugger, even though it's been called various other things in blog posts and presentations, is "Apex Debugger." We'd asked about changing the name of the existing debugger, to prevent confusion with the Replay Debugger, but thus far that request hasn't gained traction.